lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1865,20 +1865,27 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :dynamic_name => true
         },
         :'application_logging.forwarding.enabled' => {
-          :default => true,
+          :default => false,
           :public => true,
           :type => Boolean,
-          :allowed_from_server => true,
-          :description => 'If `true`, the agent captures log records emitted by your application.'
+          :allowed_from_server => false,
+          :description => 'If `true`, the agent captures log records emitted by this application.'
```

Comment:
Does the "your" -> "this" change match docs stored elsewhere? Should we update other instances of "your" such as [this one](https://docs.newrelic.com/docs/apm/agents/manage-apm-agents/app-naming/name-your-application/)?

Subjectively if it's our own description and not one belonging to the docs team, I think I'd prefer using "the".

Reply:
Great question. I changed it to `this` because I was looking at the other settings in this `default_source` file and none of them used `your`, however, I appreciate your link to the larger docs infrastructure. 

Reviewing [this article](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/) on voice, I believe we should change everything to "your" instead of "this"

Reply:
Ok cool. I like "your".